lib/samples/add_vector_tiled_layer/add_vector_tiled_layer_sample.dart

```diff
@@ -17,6 +17,27 @@
 import 'package:arcgis_maps/arcgis_maps.dart';
 import 'package:flutter/material.dart';
 
+// An enumeration of vector tiled layers to choose from.
+enum Selection {
```

Comment:
`VectorTiledItem` seems to be a better name for the `enum`

Reply:
done